lib/ipmi_backend_utils.pm

```diff
@@ -118,16 +121,23 @@ sub setup_console_in_grub {
           . "{s/(console|loglevel|log_lvl|guest_loglvl)=[^ ]*//g; "
           . "/multiboot/ s/\$/ console=com2,115200 log_lvl=all guest_loglvl=all sync_console/; "
           . "/module\\s*.*vmlinuz/ s/\$/ console=$ipmi_console,115200 console=tty loglevel=5/;}; "
-          . "s/timeout=[0-9]*/timeout=30/g;"
+          . "s/timeout=-{0,1}[0-9]{1,}/timeout=30/g;"
           . "' $grub_cfg_file";
         assert_script_run("$cmd");
         save_screenshot;
         $cmd = "sed -rn '/(multiboot|module\\s*.*vmlinuz|timeout=)/p' $grub_cfg_file";
         assert_script_run("$cmd");
+        save_screenshot;
+
+        $cmd = "sed -ri 's/^terminal.*\$/terminal_input console serial\\nterminal_output console serial\\nterminal console serial/g' $grub_cfg_file";
+        assert_script_run("$cmd");
```

Comment:
no need quote for only variable:  `assert_script_run($cmd)`

Reply:
Maybe you are right. But automatic checking does not point it out to me. I will take this into consideration and wait for Travis result.